app/teams/create/InviteForm.tsx

```
@@ -42,198 +47,291 @@
+                        {searchStats.resultCount !== 1 ? "s" : ""} found
+                      </span>
+                    </div>
+                    {filteredUsers.slice(0, 8).map((user) => (
+                      <div
+                        key={user.id}
+                        className="flex items-center gap-3 p-3 hover:bg-blue-50 cursor-pointer border-b border-gray-50 last:border-b-0 transition-colors duration-150 group"
+                        onMouseDown={(e) => {
+                          e.preventDefault();
+                        }}
+                        onClick={() => handleAddUser(user)}
+                      >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)